react-validated-input-component/my-app/src/ValidatedInput.jsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import { FaX } from 'react-icons/fa6';
+import { FaCheck } from 'react-icons/fa6';
+
+export default function ValidatedInput() {
+  const [password, setPassword] = useState('');
```

Comment:
Your solution works! However, we have a lot of state going on here! 

We can complete this challenge with just one state: 
`const [password, setPassword] = useState('');`

Reply:
A good rule of thumb, use a state variable _when a component needs to “remember” some information between renders_.